.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,9 +19,6 @@ to ensure the following steps have been taken:
 
 ### For documentation related changes:
 - [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
-- [ ] If this PR is a documentation-only change, have you added a `[doc only]`
-  tag to the first line of your commit message to avoid spending CPU cycles in
-  Travis CI when no code, tests, or build configuration are modified?
 
 ### Note:
 Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.
```

Comment:
Travis CI should be replaced by GitHub Actions

Reply:
Do we need this not anymore? The pr can't be merge with hit all checks.

Reply:
I thought the plan was to abandon Travis and use GitHub Actions only... if we need both then we can change it into "check Travis CI and GitHub Actions" here I guess.

Reply:
I'm working on the next PR already to add release tests and the docs tests to GitHub. Just for a short time we have Travis and GitHub

Reply:
maybe you can add a TODO here that you can remove when the transition is complete?

Reply:
I added a list of parts that are missing in this pr.